.github/workflows/update-icons.yml

```diff
@@ -0,0 +1,51 @@
+name: Update Icons
```

Comment:
@kim0 could you take a look at this for me?

Reply:
LGTM